apps/avifenc.c

```diff
@@ -726,7 +726,8 @@ static avifBool avifEncodeImagesFixedQuality(const avifSettings * settings,
 
         uint64_t firstDurationInTimescales = firstFile->duration ? firstFile->duration : settings->outputTiming.duration;
         if (input->useStdin || (input->filesCount > 1)) {
-            printf(" * Encoding frame 1 [%" PRIu64 "/%" PRIu64 " ts]: %s\n",
+            printf(" * Encoding frame %d [%" PRIu64 "/%" PRIu64 " ts]: %s\n",
+                   0,
```

Comment:
The other printf statement is at line 747. Since the two string literals are the same, the compiler can pool them.

Reply:
I understand. Having a literal may look a little strange and could get cleaned up at some point.